src/main/java/org/jboss/forge/arquillian/observers/DownloadServerObserver.java

```diff
@@ -1,96 +0,0 @@
-/*
- * Copyright 2013 Red Hat, Inc. and/or its affiliates.
- *
- * Licensed under the Eclipse Public License version 1.0, available at
- * http://www.eclipse.org/legal/epl-v10.html
- */
-package org.jboss.forge.arquillian.observers;
-
-import org.jboss.forge.addon.projects.Project;
-import org.jboss.forge.addon.ui.input.UIPrompt;
-
-import javax.inject.Inject;
-
-/**
- * @Author Paul Bakker - paul.bakker.nl@gmail.com
- */
-public class DownloadServerObserver {
```

Comment:
> All this logic is related to MavenProfile So I think it's good to move it into ProfileManager as per SRP.

I don't think this class follows SRP, but probably something to improve moving forward, not as a part of this PR.